services/cms/src/blocks/HeroSection/HeroSectionEditor.tsx

```diff
@@ -0,0 +1,31 @@
+import { InnerBlocks } from "@wordpress/block-editor"
+import { BlockEditProps, Template } from "@wordpress/blocks"
+import React from "react"
+
+import BlockWrapper from "../BlockWrapper"
+
+const ALLOWED_NESTED_BLOCKS = ["core/heading", "core/buttons", "core/paragraph"]
+const HERO_SECTION_TEMPLATE: Template[] = [
+  ["core/heading", { placeholder: "Welcome to...", level: 1, textAlign: "center" }],
+  ["core/paragraph", { placeholder: "Insert short sales speech...", align: "center" }],
+  [
+    "core/buttons",
```

Comment:
core/button deprecated, so if we want one button we still should use core/buttons with innerblocks.

Reply:
Hardcoded button to use shared-module primary button